scripts/bootstrap.sh

```diff
@@ -119,15 +119,18 @@ if [ -d /vagrant ]; then
     # by default the project's top.sls is disabled by file naming. hook that up here
     cd /srv/salt/ && ln -sf example.top top.sls
     
-    # install the builder base formula 
-    if [ ! -d /vagrant/cloned-projects/builder-base-formula/.git ]; then
-        git clone https://github.com/elifesciences/builder-base-formula \
-            /vagrant/cloned-projects/builder-base-formula
-    fi
-    
-    # overwrite the general purpose /etc/salt/minion file created above with 
-    # this more complex one for dev environments only
-    cp /vagrant/scripts/salt/minion /etc/salt/minion
+    # vagrant makes all formula dependencies available, including builder base formula
+
+    # overwrite the general purpose /etc/salt/minion file created above 
+    # with this more complex one for dev environments only
+    cp /vagrant/scripts/salt/minion.template /etc/salt/minion
```

Comment:
yeah I don't care very much about this, but in general I tend to limit the states of a system to as few as possible e.g. `file missing` -> `correct file present` rather than `file missing` -> `incorrect file present` -> `correct file present` or if it gets rerun during an update `correct file present` -> `incorrect file present` -> `correct file present`
(this not for performance reasons but for simplicity, e.g. a crash can leave the system in any of the intermediate state, from that it needs to recover and there are more possible inputs to consider, etc.)